discussion_agents/cog/functional/generative_agents.py

```diff
@@ -5,7 +5,7 @@
 from datetime import datetime
 from typing import Any, Dict, List, Optional, Tuple, Union
 
-import faiss
+from langchain.vectorstores import FAISS 
```

Comment:
What's the reason for this?

Reply:
because there is an import erro when i was running so i changed it.